src/components/dialog-full-screen/dialog-full-screen.stories.js

```diff
@@ -4,8 +4,10 @@ import { text, boolean } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 import { State, Store } from '@sambego/storybook-state';
 import notes from './documentation/notes.md';
-import DialogFullScreen from './dialog-full-screen';
+import Info from './documentation/Info';
```

Comment:
can we create an index file in the `documentation` and export `info` and `notes` in the index?

Reply:
Good point, added. Thanks